iphone/Classes/TiLogServer.m

```diff
@@ -223,6 +242,23 @@ +(void)startServer
 		connections = [[NSMutableArray alloc] init];
 	}
 
+	if (headers == nil) {
+		NSString* version = [NSString stringWithCString:TI_VERSION_STR encoding:NSUTF8StringEncoding];
+		NSDictionary* map = [[NSDictionary alloc] initWithObjectsAndKeys:
```

Comment:
there's a release eventually for map, which is fine. 
An alternative way you can do this is with `[NSDictionary dictionaryWithObjectsAndKeys ...]` so that you don't have to call release later.